src/sentry/api/endpoints/relay/project_configs.py

```diff
@@ -134,7 +136,7 @@ def _get_cached_or_schedule(self, public_key) -> Optional[dict]:
 
         Debouncing of the project happens after the task has been scheduled.
         """
-        cached_config = projectconfig_cache.get(public_key)
+        cached_config = projectconfig_cache.backend.get(public_key)
```

Comment:
It seems wrong to access `backend` explicitly here, should we do something like https://github.com/getsentry/sentry/blob/1d7f72ae778807e17a92f9d06e2b080f750c4b07/src/sentry/processing/realtime_metrics/__init__.py#L16-L20 instead?

Reply:
I actually want to get rid of all those hacks and make `.backend` the new pattern. `LazyServiceWrapper` is now actually typesafe, so this is more feasible and doesn't require boilerplate when creating a service.

In a followup PR I want to get rid of `expose(locals())` as well

Reply:
see https://github.com/getsentry/sentry/pull/50812

Reply:
Interesting, I've always thought of `backend` as a local helper variable, but I like the idea of getting rid of the `expose(locals())` magic. Maybe it makes sense to rename `backend` to something like `service` or `instance` after that?